src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -237,23 +263,64 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
               <Grid item xs={12}>
                 <DatePicker
                   id='startDate'
-                  label={strings.START_DATE_REQUIRED}
+                  label={record?.testType === 'Cut' ? strings.TEST_DATE_REQUIRED : strings.START_DATE_REQUIRED}
                   aria-label={strings.DATE}
                   value={record?.startDate}
                   onChange={onChangeDate}
                 />
               </Grid>
               <Grid item xs={12} marginLeft={1}>
-                <TextField
-                  label={strings.NUMBER_OF_SEEDS_TESTED_REQUIRED}
-                  type='text'
-                  onChange={onChange}
-                  id='seedsTested'
-                  value={record?.seedsTested}
-                />
+                {record?.testType === 'Cut' ? (
+                  <TextField
+                    label={strings.NUMBER_OF_SEEDS_FILLED_REQUIRED}
+                    type='text'
+                    onChange={onChangeCutValue}
+                    id='seedsFilled'
+                    value={record?.seedsFilled}
+                  />
+                ) : (
+                  <TextField
+                    label={strings.NUMBER_OF_SEEDS_TESTED_REQUIRED}
+                    type='text'
+                    onChange={onChange}
+                    id='seedsTested'
+                    value={record?.seedsTested}
+                  />
+                )}
               </Grid>
             </Box>
 
+            {record?.testType === 'Cut' && (
+              <>
+                <Grid item xs={12} display='flex'>
+                  <Grid item xs={6}>
```

Comment:
should this be 12 for mobile? similarly in other places

Reply:
Design still has 2 columns for mobile. Looks like this:
<img width="497" alt="Screen Shot 2022-09-21 at 16 18 59" src="https://user-images.githubusercontent.com/5919083/191591711-65fea037-fb6f-4023-b926-df0723bb9104.png">

Reply:
And for creating:

<img width="493" alt="Screen Shot 2022-09-21 at 16 20 15" src="https://user-images.githubusercontent.com/5919083/191591910-08c3c289-a86c-4e3e-8603-82d64bec554d.png">

Reply:
Got it, thanks for checking.